v1.12.0: The keystore properties are not valid

**Describe the bug**
The Nifi pods won't come up. 

**Version of Helm and Kubernetes**:
Helm v3.3.0
Kubernetes v1.19.2

**What happened**:
Initially installed the helm chart with some values.
The Nifi pods app-log container logs:

```
2020-09-22T12:04:54.353694909Z 2020-09-22 12:04:53,642 INFO [main] o.e.j.a.AnnotationConfiguration Scanning elapsed time=1097ms
2020-09-22T12:04:54.353732098Z 2020-09-22 12:04:53,705 INFO [main] o.e.jetty.server.handler.ContextHandler Started o.e.j.w.WebAppContext@7342e05d{/nifi-jolt-transform-json-ui-1.12.0,/nifi-jolt-transform-json-ui-1.12.0,file:///opt/nifi/nifi-current/work/jetty/nifi-jolt-transform-json-ui-1.12.0.war/webapp/,AVAILABLE}{./work/nar/extensions/nifi-standard-nar-1.12.0.nar-unpacked/NAR-INF/bundled-dependencies/nifi-jolt-transform-json-ui-1.12.0.war}
2020-09-22T12:04:54.353743265Z 2020-09-22 12:04:54,163 INFO [main] o.e.j.a.AnnotationConfiguration Scanning elapsed time=105ms
2020-09-22T12:04:54.353751273Z 2020-09-22 12:04:54,172 INFO [main] o.e.j.s.handler.ContextHandler._nifi No Spring WebApplicationInitializer types detected on classpath
2020-09-22T12:04:54.353759449Z 2020-09-22 12:04:54,243 INFO [main] o.e.jetty.server.handler.ContextHandler Started o.e.j.w.WebAppContext@2a331b46{nifi,/nifi,file:///opt/nifi/nifi-current/work/jetty/nifi-web-ui-1.12.0.war/webapp/,AVAILABLE}{./work/nar/framework/nifi-framework-nar-1.12.0.nar-unpacked/NAR-INF/bundled-dependencies/nifi-web-ui-1.12.0.war}
2020-09-22T12:04:55.353832003Z 2020-09-22 12:04:54,654 INFO [main] o.e.j.a.AnnotationConfiguration Scanning elapsed time=168ms
2020-09-22T12:04:55.353872322Z 2020-09-22 12:04:54,656 INFO [main] o.e.j.s.handler.ContextHandler._nifi_api No Spring WebApplicationInitializer types detected on classpath
2020-09-22T12:04:55.353879445Z 2020-09-22 12:04:54,752 INFO [main] o.e.j.s.handler.ContextHandler._nifi_api Initializing Spring root WebApplicationContext
2020-09-22T12:04:59.370452136Z 2020-09-22 12:04:59,042 INFO [main] org.apache.nifi.security.kms.CryptoUtils Determined default nifi.properties path to be '/opt/nifi/nifi-current/./conf/nifi.properties'
2020-09-22T12:04:59.370494327Z 2020-09-22 12:04:59,044 INFO [main] o.a.nifi.properties.NiFiPropertiesLoader Loaded 135 properties from /opt/nifi/nifi-current/./conf/nifi.properties
2020-09-22T12:05:02.385432855Z 2020-09-22 12:05:01,434 ERROR [main] org.apache.nifi.encrypt.StringEncryptor ********************************************************************************
2020-09-22T12:05:02.385472079Z 2020-09-22 12:05:01,434 ERROR [main] org.apache.nifi.encrypt.StringEncryptor *                A blank sensitive properties key was provided                 *
2020-09-22T12:05:02.385480875Z 2020-09-22 12:05:01,434 ERROR [main] org.apache.nifi.encrypt.StringEncryptor *                   Specify a unique key in nifi.properties                    *
2020-09-22T12:05:02.385488823Z 2020-09-22 12:05:01,434 ERROR [main] org.apache.nifi.encrypt.StringEncryptor *                         for nifi.sensitive.props.key                         *
2020-09-22T12:05:02.385496458Z 2020-09-22 12:05:01,434 ERROR [main] org.apache.nifi.encrypt.StringEncryptor *                                                                              *
2020-09-22T12:05:02.385518402Z 2020-09-22 12:05:01,434 ERROR [main] org.apache.nifi.encrypt.StringEncryptor *            The Encrypt Config Tool in NiFi Toolkit can be used to            *
2020-09-22T12:05:02.385526203Z 2020-09-22 12:05:01,434 ERROR [main] org.apache.nifi.encrypt.StringEncryptor *                       migrate the flow to the new key                        *
2020-09-22T12:05:02.385533796Z 2020-09-22 12:05:01,434 ERROR [main] org.apache.nifi.encrypt.StringEncryptor ********************************************************************************
2020-09-22T12:05:02.385542620Z 2020-09-22 12:05:02,030 ERROR [main] org.apache.nifi.encrypt.StringEncryptor ********************************************************************************
2020-09-22T12:05:02.385550673Z 2020-09-22 12:05:02,030 ERROR [main] org.apache.nifi.encrypt.StringEncryptor *                A blank sensitive properties key was provided                 *
2020-09-22T12:05:02.385558697Z 2020-09-22 12:05:02,030 ERROR [main] org.apache.nifi.encrypt.StringEncryptor *                   Specify a unique key in nifi.properties                    *
2020-09-22T12:05:02.385566376Z 2020-09-22 12:05:02,030 ERROR [main] org.apache.nifi.encrypt.StringEncryptor *                         for nifi.sensitive.props.key                         *
2020-09-22T12:05:02.385575484Z 2020-09-22 12:05:02,030 ERROR [main] org.apache.nifi.encrypt.StringEncryptor *                                                                              *
2020-09-22T12:05:02.385582659Z 2020-09-22 12:05:02,030 ERROR [main] org.apache.nifi.encrypt.StringEncryptor *            The Encrypt Config Tool in NiFi Toolkit can be used to            *
2020-09-22T12:05:02.385589810Z 2020-09-22 12:05:02,030 ERROR [main] org.apache.nifi.encrypt.StringEncryptor *                       migrate the flow to the new key                        *
2020-09-22T12:05:02.385596905Z 2020-09-22 12:05:02,031 ERROR [main] org.apache.nifi.encrypt.StringEncryptor ********************************************************************************
2020-09-22T12:05:02.385604070Z 2020-09-22 12:05:02,166 WARN [main] o.a.nifi.security.util.SslContextFactory Some keystore properties are populated (, null, null, JKS) but not valid
2020-09-22T12:05:02.385611677Z 2020-09-22 12:05:02,181 ERROR [main] o.a.n.c.s.m.StandardStateManagerProvider Encountered an error configuring TLS for state manager: 
2020-09-22T12:05:02.385618762Z org.apache.nifi.security.util.TlsException: The keystore properties are not valid
2020-09-22T12:05:02.385625975Z 	at org.apache.nifi.security.util.SslContextFactory.getKeyManagers(SslContextFactory.java:204)
2020-09-22T12:05:02.385633356Z 	at org.apache.nifi.security.util.SslContextFactory.createSslContext(SslContextFactory.java:131)
2020-09-22T12:05:02.385640371Z 	at org.apache.nifi.security.util.SslContextFactory.createSslContext(SslContextFactory.java:98)
2020-09-22T12:05:02.385652151Z 	at org.apache.nifi.controller.state.manager.StandardStateManagerProvider.createStateProvider(StandardStateManagerProvider.java:222)
2020-09-22T12:05:02.385659965Z 	at org.apache.nifi.controller.state.manager.StandardStateManagerProvider.createLocalStateProvider(StandardStateManagerProvider.java:105)
2020-09-22T12:05:02.385673370Z 	at org.apache.nifi.controller.state.manager.StandardStateManagerProvider.create(StandardStateManagerProvider.java:85)
2020-09-22T12:05:02.385681363Z 	at org.apache.nifi.cluster.coordination.node.NodeClusterCoordinator.<init>(NodeClusterCoordinator.java:130)
2020-09-22T12:05:02.385689078Z 	at org.apache.nifi.cluster.spring.NodeClusterCoordinatorFactoryBean.getObject(NodeClusterCoordinatorFactoryBean.java:53)
2020-09-22T12:05:02.385704476Z 	at org.apache.nifi.cluster.spring.NodeClusterCoordinatorFactoryBean.getObject(NodeClusterCoordinatorFactoryBean.java:35)
2020-09-22T12:05:02.385711969Z 	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.doGetObjectFromFactoryBean(FactoryBeanRegistrySupport.java:178)
2020-09-22T12:05:02.385720174Z 	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.getObjectFromFactoryBean(FactoryBeanRegistrySupport.java:103)
2020-09-22T12:05:02.385727802Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.getObjectForBeanInstance(AbstractBeanFactory.java:1640)
2020-09-22T12:05:02.385735039Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:323)
2020-09-22T12:05:02.385741897Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
2020-09-22T12:05:02.385749179Z 	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:351)
2020-09-22T12:05:02.385758123Z 	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:108)
2020-09-22T12:05:02.385766198Z 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyPropertyValues(AbstractAutowireCapableBeanFactory.java:1534)
2020-09-22T12:05:02.385773701Z 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1281)
2020-09-22T12:05:02.385781076Z 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:551)
2020-09-22T12:05:02.385788374Z 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:481)
2020-09-22T12:05:02.385795821Z 	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:312)
2020-09-22T12:05:02.385802635Z 	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
2020-09-22T12:05:02.385809892Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:308)
2020-09-22T12:05:02.385817169Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
2020-09-22T12:05:02.385824149Z 	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1086)
2020-09-22T12:05:02.385836898Z 	at org.apache.nifi.spring.StandardFlowServiceFactoryBean.getObject(StandardFlowServiceFactoryBean.java:48)
2020-09-22T12:05:02.385845123Z 	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.doGetObjectFromFactoryBean(FactoryBeanRegistrySupport.java:178)
2020-09-22T12:05:02.385858960Z 	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.getObjectFromFactoryBean(FactoryBeanRegistrySupport.java:103)
2020-09-22T12:05:02.385866070Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.getObjectForBeanInstance(AbstractBeanFactory.java:1640)
2020-09-22T12:05:02.385872850Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:323)
2020-09-22T12:05:02.385880095Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
2020-09-22T12:05:02.385886891Z 	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1086)
2020-09-22T12:05:02.385894160Z 	at org.apache.nifi.web.contextlistener.ApplicationStartupContextListener.contextInitialized(ApplicationStartupContextListener.java:55)
2020-09-22T12:05:02.385901794Z 	at org.eclipse.jetty.server.handler.ContextHandler.callContextInitialized(ContextHandler.java:930)
2020-09-22T12:05:02.385908861Z 	at org.eclipse.jetty.servlet.ServletContextHandler.callContextInitialized(ServletContextHandler.java:553)
2020-09-22T12:05:02.385915762Z 	at org.eclipse.jetty.server.handler.ContextHandler.startContext(ContextHandler.java:889)
2020-09-22T12:05:02.385922509Z 	at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:356)
2020-09-22T12:05:02.385929231Z 	at org.eclipse.jetty.webapp.WebAppContext.startWebapp(WebAppContext.java:1445)
2020-09-22T12:05:02.385936215Z 	at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1409)
2020-09-22T12:05:02.385943168Z 	at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:822)
2020-09-22T12:05:02.385950015Z 	at org.eclipse.jetty.servlet.ServletContextHandler.doStart(ServletContextHandler.java:275)
2020-09-22T12:05:02.385956723Z 	at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:524)
2020-09-22T12:05:02.385963580Z 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
2020-09-22T12:05:02.385970382Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
2020-09-22T12:05:02.385977477Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
2020-09-22T12:05:02.385984304Z 	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
2020-09-22T12:05:02.385991164Z 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
2020-09-22T12:05:02.385998556Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
2020-09-22T12:05:02.386005320Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:110)
2020-09-22T12:05:02.386023861Z 	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
2020-09-22T12:05:02.386031301Z 	at org.eclipse.jetty.server.handler.gzip.GzipHandler.doStart(GzipHandler.java:425)
2020-09-22T12:05:02.386041650Z 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
2020-09-22T12:05:02.386048554Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
2020-09-22T12:05:02.386055757Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
2020-09-22T12:05:02.386062992Z 	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
2020-09-22T12:05:02.386075195Z 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
2020-09-22T12:05:02.386082179Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
2020-09-22T12:05:02.386088967Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
2020-09-22T12:05:02.386095724Z 	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
2020-09-22T12:05:02.386103443Z 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
2020-09-22T12:05:02.386110230Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
2020-09-22T12:05:02.386117553Z 	at org.eclipse.jetty.server.Server.start(Server.java:407)
2020-09-22T12:05:02.386124368Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:110)
2020-09-22T12:05:02.386131263Z 	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
2020-09-22T12:05:02.386138089Z 	at org.eclipse.jetty.server.Server.doStart(Server.java:371)
2020-09-22T12:05:02.386145153Z 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
2020-09-22T12:05:02.386152297Z 	at org.apache.nifi.web.server.JettyServer.start(JettyServer.java:1058)
2020-09-22T12:05:02.386159047Z 	at org.apache.nifi.NiFi.<init>(NiFi.java:158)
2020-09-22T12:05:02.386165953Z 	at org.apache.nifi.NiFi.<init>(NiFi.java:72)
2020-09-22T12:05:02.386175169Z 	at org.apache.nifi.NiFi.main(NiFi.java:301)
2020-09-22T12:05:02.386182900Z 2020-09-22 12:05:02,185 WARN [main] org.eclipse.jetty.webapp.WebAppContext Failed startup of context o.e.j.w.WebAppContext@15383681{nifi-api,/nifi-api,file:///opt/nifi/nifi-current/work/jetty/nifi-web-api-1.12.0.war/webapp/,UNAVAILABLE}{./work/nar/framework/nifi-framework-nar-1.12.0.nar-unpacked/NAR-INF/bundled-dependencies/nifi-web-api-1.12.0.war}
2020-09-22T12:05:02.386191491Z org.apache.nifi.web.NiFiCoreException: Unable to start Flow Controller.
2020-09-22T12:05:02.386198745Z 	at org.apache.nifi.web.contextlistener.ApplicationStartupContextListener.contextInitialized(ApplicationStartupContextListener.java:88)
2020-09-22T12:05:02.386211250Z 	at org.eclipse.jetty.server.handler.ContextHandler.callContextInitialized(ContextHandler.java:930)
2020-09-22T12:05:02.386218529Z 	at org.eclipse.jetty.servlet.ServletContextHandler.callContextInitialized(ServletContextHandler.java:553)
2020-09-22T12:05:02.386225618Z 	at org.eclipse.jetty.server.handler.ContextHandler.startContext(ContextHandler.java:889)
2020-09-22T12:05:02.386232405Z 	at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:356)
2020-09-22T12:05:02.386241193Z 	at org.eclipse.jetty.webapp.WebAppContext.startWebapp(WebAppContext.java:1445)
2020-09-22T12:05:02.386254999Z 	at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1409)
2020-09-22T12:05:02.386265036Z 	at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:822)
2020-09-22T12:05:02.386271828Z 	at org.eclipse.jetty.servlet.ServletContextHandler.doStart(ServletContextHandler.java:275)
2020-09-22T12:05:02.386278554Z 	at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:524)
2020-09-22T12:05:02.386286973Z 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
2020-09-22T12:05:02.386312416Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
2020-09-22T12:05:02.386321796Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
2020-09-22T12:05:02.386328695Z 	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
2020-09-22T12:05:02.386335330Z 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
2020-09-22T12:05:02.386341955Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
2020-09-22T12:05:02.386348760Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:110)
2020-09-22T12:05:02.386355475Z 	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
2020-09-22T12:05:02.386362104Z 	at org.eclipse.jetty.server.handler.gzip.GzipHandler.doStart(GzipHandler.java:425)
2020-09-22T12:05:02.386368769Z 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
2020-09-22T12:05:02.386376306Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
2020-09-22T12:05:02.386383215Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
2020-09-22T12:05:02.386389904Z 	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
2020-09-22T12:05:02.386396531Z 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
2020-09-22T12:05:02.386403315Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
2020-09-22T12:05:02.386409916Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
2020-09-22T12:05:02.386421735Z 	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
2020-09-22T12:05:02.386428723Z 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
2020-09-22T12:05:02.386435642Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
2020-09-22T12:05:02.386442710Z 	at org.eclipse.jetty.server.Server.start(Server.java:407)
2020-09-22T12:05:02.386449434Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:110)
2020-09-22T12:05:02.386456619Z 	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
2020-09-22T12:05:02.386463709Z 	at org.eclipse.jetty.server.Server.doStart(Server.java:371)
2020-09-22T12:05:02.386470528Z 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
2020-09-22T12:05:02.386477651Z 	at org.apache.nifi.web.server.JettyServer.start(JettyServer.java:1058)
2020-09-22T12:05:02.386484843Z 	at org.apache.nifi.NiFi.<init>(NiFi.java:158)
2020-09-22T12:05:02.386491950Z 	at org.apache.nifi.NiFi.<init>(NiFi.java:72)
2020-09-22T12:05:02.386499849Z 	at org.apache.nifi.NiFi.main(NiFi.java:301)
2020-09-22T12:05:02.386508083Z Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'flowService': FactoryBean threw exception on object creation; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'flowController' defined in class path resource [nifi-context.xml]: Cannot resolve reference to bean 'clusterCoordinator' while setting bean property 'clusterCoordinator'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'clusterCoordinator': FactoryBean threw exception on object creation; nested exception is java.lang.IllegalStateException: Error configuring TLS for state manager
2020-09-22T12:05:02.386518875Z 	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.doGetObjectFromFactoryBean(FactoryBeanRegistrySupport.java:185)
2020-09-22T12:05:02.386526200Z 	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.getObjectFromFactoryBean(FactoryBeanRegistrySupport.java:103)
2020-09-22T12:05:02.386533645Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.getObjectForBeanInstance(AbstractBeanFactory.java:1640)
2020-09-22T12:05:02.386540828Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:323)
2020-09-22T12:05:02.386547794Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
2020-09-22T12:05:02.386555081Z 	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1086)
2020-09-22T12:05:02.386563832Z 	at org.apache.nifi.web.contextlistener.ApplicationStartupContextListener.contextInitialized(ApplicationStartupContextListener.java:55)
2020-09-22T12:05:02.386571345Z 	... 37 common frames omitted
2020-09-22T12:05:02.386577933Z Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'flowController' defined in class path resource [nifi-context.xml]: Cannot resolve reference to bean 'clusterCoordinator' while setting bean property 'clusterCoordinator'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'clusterCoordinator': FactoryBean threw exception on object creation; nested exception is java.lang.IllegalStateException: Error configuring TLS for state manager
2020-09-22T12:05:02.386597848Z 	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:359)
2020-09-22T12:05:02.386606236Z 	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:108)
2020-09-22T12:05:02.386613835Z 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyPropertyValues(AbstractAutowireCapableBeanFactory.java:1534)
2020-09-22T12:05:02.386621539Z 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1281)
2020-09-22T12:05:02.386629922Z 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:551)
2020-09-22T12:05:02.386637348Z 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:481)
2020-09-22T12:05:02.386644698Z 	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:312)
2020-09-22T12:05:02.386651729Z 	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
2020-09-22T12:05:02.386658761Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:308)
2020-09-22T12:05:02.386666318Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
2020-09-22T12:05:02.386673221Z 	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1086)
2020-09-22T12:05:02.386680030Z 	at org.apache.nifi.spring.StandardFlowServiceFactoryBean.getObject(StandardFlowServiceFactoryBean.java:48)
2020-09-22T12:05:02.386687085Z 	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.doGetObjectFromFactoryBean(FactoryBeanRegistrySupport.java:178)
2020-09-22T12:05:02.386695213Z 	... 43 common frames omitted
2020-09-22T12:05:02.386701946Z Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'clusterCoordinator': FactoryBean threw exception on object creation; nested exception is java.lang.IllegalStateException: Error configuring TLS for state manager
2020-09-22T12:05:02.386709733Z 	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.doGetObjectFromFactoryBean(FactoryBeanRegistrySupport.java:185)
2020-09-22T12:05:02.386716950Z 	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.getObjectFromFactoryBean(FactoryBeanRegistrySupport.java:103)
2020-09-22T12:05:02.386727779Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.getObjectForBeanInstance(AbstractBeanFactory.java:1640)
2020-09-22T12:05:02.386740686Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:323)
2020-09-22T12:05:02.386747774Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
2020-09-22T12:05:02.386754839Z 	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:351)
2020-09-22T12:05:02.386763254Z 	... 55 common frames omitted
2020-09-22T12:05:02.386770183Z Caused by: java.lang.IllegalStateException: Error configuring TLS for state manager
2020-09-22T12:05:02.386777310Z 	at org.apache.nifi.controller.state.manager.StandardStateManagerProvider.createStateProvider(StandardStateManagerProvider.java:225)
2020-09-22T12:05:02.386787173Z 	at org.apache.nifi.controller.state.manager.StandardStateManagerProvider.createLocalStateProvider(StandardStateManagerProvider.java:105)
2020-09-22T12:05:02.386795029Z 	at org.apache.nifi.controller.state.manager.StandardStateManagerProvider.create(StandardStateManagerProvider.java:85)
2020-09-22T12:05:02.386803721Z 	at org.apache.nifi.cluster.coordination.node.NodeClusterCoordinator.<init>(NodeClusterCoordinator.java:130)
2020-09-22T12:05:02.386811349Z 	at org.apache.nifi.cluster.spring.NodeClusterCoordinatorFactoryBean.getObject(NodeClusterCoordinatorFactoryBean.java:53)
2020-09-22T12:05:02.386819022Z 	at org.apache.nifi.cluster.spring.NodeClusterCoordinatorFactoryBean.getObject(NodeClusterCoordinatorFactoryBean.java:35)
2020-09-22T12:05:02.386826029Z 	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.doGetObjectFromFactoryBean(FactoryBeanRegistrySupport.java:178)
2020-09-22T12:05:02.386833051Z 	... 60 common frames omitted
2020-09-22T12:05:02.386839904Z Caused by: org.apache.nifi.security.util.TlsException: The keystore properties are not valid
2020-09-22T12:05:02.386846814Z 	at org.apache.nifi.security.util.SslContextFactory.getKeyManagers(SslContextFactory.java:204)
2020-09-22T12:05:02.386853511Z 	at org.apache.nifi.security.util.SslContextFactory.createSslContext(SslContextFactory.java:131)
2020-09-22T12:05:02.386860436Z 	at org.apache.nifi.security.util.SslContextFactory.createSslContext(SslContextFactory.java:98)
2020-09-22T12:05:02.386867420Z 	at org.apache.nifi.controller.state.manager.StandardStateManagerProvider.createStateProvider(StandardStateManagerProvider.java:222)
2020-09-22T12:05:02.386874839Z 	... 66 common frames omitted
2020-09-22T12:05:03.378331270Z 2020-09-22 12:05:02,572 INFO [main] o.e.j.a.AnnotationConfiguration Scanning elapsed time=116ms
2020-09-22T12:05:03.378365182Z 2020-09-22 12:05:02,574 INFO [main] o.e.j.s.h.C._nifi_content_viewer No Spring WebApplicationInitializer types detected on classpath
2020-09-22T12:05:03.378372362Z 2020-09-22 12:05:02,605 INFO [main] o.e.jetty.server.handler.ContextHandler Started o.e.j.w.WebAppContext@743e66f7{nifi-content-viewer,/nifi-content-viewer,file:///opt/nifi/nifi-current/work/jetty/nifi-web-content-viewer-1.12.0.war/webapp/,AVAILABLE}{./work/nar/framework/nifi-framework-nar-1.12.0.nar-unpacked/NAR-INF/bundled-dependencies/nifi-web-content-viewer-1.12.0.war}
2020-09-22T12:05:03.378392246Z 2020-09-22 12:05:02,639 INFO [main] o.e.j.a.AnnotationConfiguration Scanning elapsed time=14ms
2020-09-22T12:05:03.378399075Z 2020-09-22 12:05:02,641 WARN [main] o.e.j.webapp.StandardDescriptorProcessor Duplicate mapping from / to default
2020-09-22T12:05:03.378405326Z 2020-09-22 12:05:02,642 INFO [main] o.e.j.s.h.ContextHandler._nifi_docs No Spring WebApplicationInitializer types detected on classpath
2020-09-22T12:05:03.378411891Z 2020-09-22 12:05:02,691 INFO [main] o.e.jetty.server.handler.ContextHandler Started o.e.j.w.WebAppContext@109a2025{nifi-docs,/nifi-docs,file:///opt/nifi/nifi-current/work/jetty/nifi-web-docs-1.12.0.war/webapp/,AVAILABLE}{./work/nar/framework/nifi-framework-nar-1.12.0.nar-unpacked/NAR-INF/bundled-dependencies/nifi-web-docs-1.12.0.war}
2020-09-22T12:05:03.378420153Z 2020-09-22 12:05:02,858 INFO [main] o.e.j.a.AnnotationConfiguration Scanning elapsed time=129ms
2020-09-22T12:05:03.378425917Z 2020-09-22 12:05:02,878 INFO [main] o.e.j.server.handler.ContextHandler._ No Spring WebApplicationInitializer types detected on classpath
2020-09-22T12:05:03.378431926Z 2020-09-22 12:05:02,912 INFO [main] o.e.jetty.server.handler.ContextHandler Started o.e.j.w.WebAppContext@2241f05b{nifi-error,/,file:///opt/nifi/nifi-current/work/jetty/nifi-web-error-1.12.0.war/webapp/,AVAILABLE}{./work/nar/framework/nifi-framework-nar-1.12.0.nar-unpacked/NAR-INF/bundled-dependencies/nifi-web-error-1.12.0.war}
2020-09-22T12:05:03.378438655Z 2020-09-22 12:05:02,961 INFO [main] o.eclipse.jetty.server.AbstractConnector Started ServerConnector@67a3bd51{HTTP/1.1,[http/1.1]}{10.233.116.6:8080}
2020-09-22T12:05:03.378446151Z 2020-09-22 12:05:02,963 INFO [main] o.eclipse.jetty.server.AbstractConnector Started ServerConnector@3ce53f6a{HTTP/1.1,[http/1.1]}{127.0.0.1:8080}
2020-09-22T12:05:03.378452529Z 2020-09-22 12:05:02,964 INFO [main] org.eclipse.jetty.server.Server Started @179497ms
2020-09-22T12:05:03.378458325Z 2020-09-22 12:05:02,964 WARN [main] org.apache.nifi.web.server.JettyServer Failed to start web server... shutting down.
2020-09-22T12:05:03.378464172Z org.apache.nifi.web.NiFiCoreException: Unable to start Flow Controller.
2020-09-22T12:05:03.378469871Z 	at org.apache.nifi.web.contextlistener.ApplicationStartupContextListener.contextInitialized(ApplicationStartupContextListener.java:88)
2020-09-22T12:05:03.378476074Z 	at org.eclipse.jetty.server.handler.ContextHandler.callContextInitialized(ContextHandler.java:930)
2020-09-22T12:05:03.378483137Z 	at org.eclipse.jetty.servlet.ServletContextHandler.callContextInitialized(ServletContextHandler.java:553)
2020-09-22T12:05:03.378495094Z 	at org.eclipse.jetty.server.handler.ContextHandler.startContext(ContextHandler.java:889)
2020-09-22T12:05:03.378503005Z 	at org.eclipse.jetty.servlet.ServletContextHandler.startContext(ServletContextHandler.java:356)
2020-09-22T12:05:03.378511613Z 	at org.eclipse.jetty.webapp.WebAppContext.startWebapp(WebAppContext.java:1445)
2020-09-22T12:05:03.378519152Z 	at org.eclipse.jetty.webapp.WebAppContext.startContext(WebAppContext.java:1409)
2020-09-22T12:05:03.378526044Z 	at org.eclipse.jetty.server.handler.ContextHandler.doStart(ContextHandler.java:822)
2020-09-22T12:05:03.378541995Z 	at org.eclipse.jetty.servlet.ServletContextHandler.doStart(ServletContextHandler.java:275)
2020-09-22T12:05:03.378551760Z 	at org.eclipse.jetty.webapp.WebAppContext.doStart(WebAppContext.java:524)
2020-09-22T12:05:03.378563718Z 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
2020-09-22T12:05:03.378571042Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
2020-09-22T12:05:03.378577190Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
2020-09-22T12:05:03.378583066Z 	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
2020-09-22T12:05:03.378588803Z 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
2020-09-22T12:05:03.378594482Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
2020-09-22T12:05:03.378600185Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:110)
2020-09-22T12:05:03.378605889Z 	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
2020-09-22T12:05:03.378612458Z 	at org.eclipse.jetty.server.handler.gzip.GzipHandler.doStart(GzipHandler.java:425)
2020-09-22T12:05:03.378618171Z 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
2020-09-22T12:05:03.378623922Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
2020-09-22T12:05:03.378629678Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
2020-09-22T12:05:03.378635655Z 	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
2020-09-22T12:05:03.378641368Z 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
2020-09-22T12:05:03.378647144Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
2020-09-22T12:05:03.378652839Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:117)
2020-09-22T12:05:03.378658616Z 	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
2020-09-22T12:05:03.378664285Z 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
2020-09-22T12:05:03.378670029Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.start(ContainerLifeCycle.java:169)
2020-09-22T12:05:03.378675736Z 	at org.eclipse.jetty.server.Server.start(Server.java:407)
2020-09-22T12:05:03.378681368Z 	at org.eclipse.jetty.util.component.ContainerLifeCycle.doStart(ContainerLifeCycle.java:110)
2020-09-22T12:05:03.378687037Z 	at org.eclipse.jetty.server.handler.AbstractHandler.doStart(AbstractHandler.java:97)
2020-09-22T12:05:03.378692653Z 	at org.eclipse.jetty.server.Server.doStart(Server.java:371)
2020-09-22T12:05:03.378703940Z 	at org.eclipse.jetty.util.component.AbstractLifeCycle.start(AbstractLifeCycle.java:72)
2020-09-22T12:05:03.378710059Z 	at org.apache.nifi.web.server.JettyServer.start(JettyServer.java:1058)
2020-09-22T12:05:03.378715782Z 	at org.apache.nifi.NiFi.<init>(NiFi.java:158)
2020-09-22T12:05:03.378721645Z 	at org.apache.nifi.NiFi.<init>(NiFi.java:72)
2020-09-22T12:05:03.378727514Z 	at org.apache.nifi.NiFi.main(NiFi.java:301)
2020-09-22T12:05:03.378734403Z Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'flowService': FactoryBean threw exception on object creation; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'flowController' defined in class path resource [nifi-context.xml]: Cannot resolve reference to bean 'clusterCoordinator' while setting bean property 'clusterCoordinator'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'clusterCoordinator': FactoryBean threw exception on object creation; nested exception is java.lang.IllegalStateException: Error configuring TLS for state manager
2020-09-22T12:05:03.378742678Z 	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.doGetObjectFromFactoryBean(FactoryBeanRegistrySupport.java:185)
2020-09-22T12:05:03.378748611Z 	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.getObjectFromFactoryBean(FactoryBeanRegistrySupport.java:103)
2020-09-22T12:05:03.378754523Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.getObjectForBeanInstance(AbstractBeanFactory.java:1640)
2020-09-22T12:05:03.378760451Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:323)
2020-09-22T12:05:03.378766262Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
2020-09-22T12:05:03.378773198Z 	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1086)
2020-09-22T12:05:03.378781069Z 	at org.apache.nifi.web.contextlistener.ApplicationStartupContextListener.contextInitialized(ApplicationStartupContextListener.java:55)
2020-09-22T12:05:03.378788664Z 	... 37 common frames omitted
2020-09-22T12:05:03.378797065Z Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'flowController' defined in class path resource [nifi-context.xml]: Cannot resolve reference to bean 'clusterCoordinator' while setting bean property 'clusterCoordinator'; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'clusterCoordinator': FactoryBean threw exception on object creation; nested exception is java.lang.IllegalStateException: Error configuring TLS for state manager
2020-09-22T12:05:03.378811957Z 	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:359)
2020-09-22T12:05:03.378819833Z 	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveValueIfNecessary(BeanDefinitionValueResolver.java:108)
2020-09-22T12:05:03.378828034Z 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.applyPropertyValues(AbstractAutowireCapableBeanFactory.java:1534)
2020-09-22T12:05:03.378842867Z 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.populateBean(AbstractAutowireCapableBeanFactory.java:1281)
2020-09-22T12:05:03.378850902Z 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.doCreateBean(AbstractAutowireCapableBeanFactory.java:551)
2020-09-22T12:05:03.378858246Z 	at org.springframework.beans.factory.support.AbstractAutowireCapableBeanFactory.createBean(AbstractAutowireCapableBeanFactory.java:481)
2020-09-22T12:05:03.378864439Z 	at org.springframework.beans.factory.support.AbstractBeanFactory$1.getObject(AbstractBeanFactory.java:312)
2020-09-22T12:05:03.378870327Z 	at org.springframework.beans.factory.support.DefaultSingletonBeanRegistry.getSingleton(DefaultSingletonBeanRegistry.java:230)
2020-09-22T12:05:03.378877953Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:308)
2020-09-22T12:05:03.378885567Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:202)
2020-09-22T12:05:03.378892763Z 	at org.springframework.context.support.AbstractApplicationContext.getBean(AbstractApplicationContext.java:1086)
2020-09-22T12:05:03.378900340Z 	at org.apache.nifi.spring.StandardFlowServiceFactoryBean.getObject(StandardFlowServiceFactoryBean.java:48)
2020-09-22T12:05:03.378907574Z 	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.doGetObjectFromFactoryBean(FactoryBeanRegistrySupport.java:178)
2020-09-22T12:05:03.378914564Z 	... 43 common frames omitted
2020-09-22T12:05:03.378922307Z Caused by: org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'clusterCoordinator': FactoryBean threw exception on object creation; nested exception is java.lang.IllegalStateException: Error configuring TLS for state manager
2020-09-22T12:05:03.378928835Z 	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.doGetObjectFromFactoryBean(FactoryBeanRegistrySupport.java:185)
2020-09-22T12:05:03.378934751Z 	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.getObjectFromFactoryBean(FactoryBeanRegistrySupport.java:103)
2020-09-22T12:05:03.378940686Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.getObjectForBeanInstance(AbstractBeanFactory.java:1640)
2020-09-22T12:05:03.378946476Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.doGetBean(AbstractBeanFactory.java:323)
2020-09-22T12:05:03.378953055Z 	at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:197)
2020-09-22T12:05:03.378958957Z 	at org.springframework.beans.factory.support.BeanDefinitionValueResolver.resolveReference(BeanDefinitionValueResolver.java:351)
2020-09-22T12:05:03.378964750Z 	... 55 common frames omitted
2020-09-22T12:05:03.378970260Z Caused by: java.lang.IllegalStateException: Error configuring TLS for state manager
2020-09-22T12:05:03.378975881Z 	at org.apache.nifi.controller.state.manager.StandardStateManagerProvider.createStateProvider(StandardStateManagerProvider.java:225)
2020-09-22T12:05:03.378987273Z 	at org.apache.nifi.controller.state.manager.StandardStateManagerProvider.createLocalStateProvider(StandardStateManagerProvider.java:105)
2020-09-22T12:05:03.378993470Z 	at org.apache.nifi.controller.state.manager.StandardStateManagerProvider.create(StandardStateManagerProvider.java:85)
2020-09-22T12:05:03.378999304Z 	at org.apache.nifi.cluster.coordination.node.NodeClusterCoordinator.<init>(NodeClusterCoordinator.java:130)
2020-09-22T12:05:03.379005361Z 	at org.apache.nifi.cluster.spring.NodeClusterCoordinatorFactoryBean.getObject(NodeClusterCoordinatorFactoryBean.java:53)
2020-09-22T12:05:03.379011165Z 	at org.apache.nifi.cluster.spring.NodeClusterCoordinatorFactoryBean.getObject(NodeClusterCoordinatorFactoryBean.java:35)
2020-09-22T12:05:03.379016976Z 	at org.springframework.beans.factory.support.FactoryBeanRegistrySupport.doGetObjectFromFactoryBean(FactoryBeanRegistrySupport.java:178)
2020-09-22T12:05:03.379022850Z 	... 60 common frames omitted
2020-09-22T12:05:03.379028358Z Caused by: org.apache.nifi.security.util.TlsException: The keystore properties are not valid
2020-09-22T12:05:03.379033948Z 	at org.apache.nifi.security.util.SslContextFactory.getKeyManagers(SslContextFactory.java:204)
2020-09-22T12:05:03.379039598Z 	at org.apache.nifi.security.util.SslContextFactory.createSslContext(SslContextFactory.java:131)
2020-09-22T12:05:03.379045317Z 	at org.apache.nifi.security.util.SslContextFactory.createSslContext(SslContextFactory.java:98)
2020-09-22T12:05:03.379051077Z 	at org.apache.nifi.controller.state.manager.StandardStateManagerProvider.createStateProvider(StandardStateManagerProvider.java:222)
2020-09-22T12:05:03.379056879Z 	... 66 common frames omitted
2020-09-22T12:05:03.379062419Z 2020-09-22 12:05:02,969 INFO [Thread-1] org.apache.nifi.NiFi Initiating shutdown of Jetty web server...
2020-09-22T12:05:03.379068104Z 2020-09-22 12:05:02,980 INFO [Thread-1] o.eclipse.jetty.server.AbstractConnector Stopped ServerConnector@67a3bd51{HTTP/1.1,[http/1.1]}{10.233.116.6:8080}
2020-09-22T12:05:03.379074654Z 2020-09-22 12:05:02,984 INFO [Thread-1] o.eclipse.jetty.server.AbstractConnector Stopped ServerConnector@3ce53f6a{HTTP/1.1,[http/1.1]}{127.0.0.1:8080}
2020-09-22T12:05:03.379080596Z 2020-09-22 12:05:02,984 INFO [Thread-1] org.eclipse.jetty.server.session node0 Stopped scavenging
```


This makes Jetty shutdown. And the pod gets int CrashLoopBackoff

**What you expected to happen**:
Everything is deployed and application runs flawlessly.

**How to reproduce it** (as minimally and precisely as possible):

Config values
* using `externalSecure` set to `true`
* CA not enabled
* persistent storage turned on
* Service is `LoadBalancer`
* Use image tag `1.12.0` (I guess that's the reason?!)

Then
* `helm upgrade -i nifi cetic/nifi -f nifi-values.yaml`

--- a/Chart.yaml
+++ b/Chart.yaml
@@ -1,7 +1,7 @@
 ---
 apiVersion: v2
 name: nifi
-version: 0.5.4
+version: 0.5.5
 appVersion: 1.12.1
 description: Apache NiFi is a software project from the Apache Software Foundation designed to automate the flow of data between software systems.
 keywords:

--- a/configs/nifi.properties
+++ b/configs/nifi.properties
@@ -151,11 +151,11 @@ nifi.sensitive.props.provider=BC
 nifi.sensitive.props.additional.keys=
 
 nifi.security.keystore=
-nifi.security.keystoreType=jks
+nifi.security.keystoreType=
 nifi.security.keystorePasswd=
 nifi.security.keyPasswd=
 nifi.security.truststore=
-nifi.security.truststoreType=jks
+nifi.security.truststoreType=
 nifi.security.truststorePasswd=
 nifi.security.needClientAuth={{.Values.properties.needClientAuth}}
 nifi.security.user.authorizer={{.Values.properties.authorizer}}

--- a/templates/ingress.yaml
+++ b/templates/ingress.yaml
@@ -2,7 +2,11 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "apache-nifi.fullname" . -}}
 {{- $ingressPath := .Values.ingress.path -}}
+{{- if .Values.properties.clusterSecure -}}
+{{- $ingressPort := .Values.service.httpsPort -}}
+{{- else }}
 {{- $ingressPort := .Values.service.httpPort -}}
+{{- end }}
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:

--- a/templates/statefulset.yaml
+++ b/templates/statefulset.yaml
@@ -188,9 +188,11 @@ spec:
 {{- if .Values.properties.clusterSecure }}
           # Update nifi.properties for security properties
           prop_replace nifi.web.https.host ${FQDN}
+          prop_replace nifi.security.keystoreType jks
           prop_replace nifi.security.keystore   ${NIFI_HOME}/config-data/certs/keystore.jks
           prop_replace nifi.security.keystorePasswd     $(jq -r .keyStorePassword ${NIFI_HOME}/config-data/certs/config.json)
           prop_replace nifi.security.keyPasswd          $(jq -r .keyPassword ${NIFI_HOME}/config-data/certs/config.json)
+          prop_replace nifi.security.truststoreType jks
           prop_replace nifi.security.truststore   ${NIFI_HOME}/config-data/certs/truststore.jks
           prop_replace nifi.security.truststorePasswd   $(jq -r .trustStorePassword ${NIFI_HOME}/config-data/certs/config.json)
 

--- a/templates/statefulset.yaml
+++ b/templates/statefulset.yaml
@@ -50,7 +50,7 @@ spec:
                podAffinityTerm:
                  labelSelector:
                     matchExpressions:
-                      - key: "component"
+                      - key: "app"
                         operator: In
                         values:
                          - {{ include "apache-nifi.name" . | quote }}

